src/main/java/net/mcreator/ui/help/HelpLoader.java

```diff
@@ -92,8 +92,7 @@ public static String loadHelpFor(IHelpContext helpContext, @Nullable Supplier<?>
 				if (helpText != null) {
 					if (contextArguments != null)

```

Comment:
This check checks for null so `() -> ""` should't be needed and also must not be as this is smelly code as supplying one empty string is not really descriptive and wrong

Reply:
Should I change it to `() -> null`? Because null Supplier variable returns _NullPointerException_ when opening a help tip (the code gets a value provided by Supplier, how can it execute `null.get()`?).

Reply:
Check for null value to prevent calling null.get. Supplier can be null, just null check it

Reply:
I guess it's just fixed at a wrong line, I'll resolve that tonight.

Reply:
I got what you meant, will resolve that tonight.

Reply:
I have resolved this in master just now as it was a really easy fix ;)